featuretools/tests/plugin_tests/test_plugin.py

```diff
@@ -0,0 +1,16 @@
+from featuretools.tests.plugin_tests.utils import (
+    import_featuretools,
+    install_featuretools_plugin,
+    reinstall_pandas,
+    uninstall_featuretools_plugin
+)
+
+
+def test_plugin_warning():
+    install_featuretools_plugin()
+    reinstall_pandas()
```

Comment:
While package version compatibility is an actual way entrypoints have failed in the past, I think we can simplify this test if we avoid the reinstalling packages and instead have the featuretools_plugin package raise an error in it's __init__ .py file so it always errors.  That way we don't need to reinstall pandas